area/area.go

```diff
@@ -135,3 +141,57 @@ func (m *GormAreaRepository) ListChildren(ctx context.Context, parentArea *Area)
 	}
 	return objs, nil
 }
+
+// Root fetches the Root Areas inside a space.
```

Comment:
Just want to avoid misunderstandings in future while reading the comment.
`Areas inside` to `Area inside` :)